diffs/packetbeatdiff.diff

```diff
@@ -0,0 +1,2602 @@
+html/en/beats/packetbeat/master/add-cloud-metadata.html
+html/en/beats/packetbeat/master/add-docker-metadata.html
+html/en/beats/packetbeat/master/add-fields.html
+html/en/beats/packetbeat/master/add-host-metadata.html
+html/en/beats/packetbeat/master/add-kubernetes-metadata.html
+html/en/beats/packetbeat/master/add-labels.html
+--- /dev/fd/63
++++ html/en/beats/packetbeat/master/add-labels.html
+@@ -214,8 +214,8 @@
+         with.dots: nested
+       array:
+         - do
+-        - re

```

Comment:
Fixed in https://github.com/elastic/beats/pull/13722/commits/89f38283eef6e2d5bb45c4f156949168bf41dabd